tests/unit/expire.tcl

```diff
@@ -854,8 +854,7 @@ start_cluster 1 0 {tags {"expire external:skip cluster slow"}} {
         r psetex key 500 val
 
         # disable resizing
-        r config set rdb-key-save-delay 10000000
-        r bgsave
+        r debug dict-resizing 0
```

Comment:
let's add a comment that explains that we use this method and not a slow BGSAVE, because this test passes the dict_force_resize_ratio threshold.

and do the same in other tests.

Reply:
I'm actually thinking of just using this to disable resize in the future (instead of using slow bgsave to disable it).

but i agree this is better, to differentiate them, and we have force